### PR TITLE
chore: Add new plugin metadata

### PR DESCRIPTION
**Pull request by suddjian**
💔 Breaking Changes

🏆 Enhancements

Adds example images, categories, descriptions, and tags to some of the plugins

📜 Documentation

🐛 Bug Fix

🏠 Internal

### CODE REVIEW

File: plugins/legacy-plugin-chart-calendar/src/index.js [hunk 1]
```diff
@@ -22,8 +22,11 @@ import controlPanel from './controlPanel';
 import thumbnail from './images/thumbnail.png';
 
 const metadata = new ChartMetadata({
+  category: 'Time Series',
```
Comment (rusackas) [inline plugins/legacy-plugin-chart-calendar/src/index.js:25]: t()

Comment (suddjian) [inline plugins/legacy-plugin-chart-calendar/src/index.js:25]:
```suggestion
  category: t('Time Series'),
```

File: plugins/legacy-preset-chart-big-number/src/BigNumber/index.ts [hunk 2]
```diff
@@ -19,10 +19,19 @@
...
 
 const metadata = new ChartMetadata({
-  description: '',
+  category: t('Time Series'),
+  description: t(
+    'A single number accompanied by a simple line chart, to call attention to an important metric along with its change over time or other dimension. If you find yourself using a lot of big numbers on a dashboard, a more visual chart may help you make your dashboard more informative.',
+  ),
+  exampleGallery: [
+    { url: example1, caption: 'A Big Number' },
+    { url: example2, caption: 'With a titlee' },
```
Comment (suddjian) [inline plugins/legacy-preset-chart-big-number/src/BigNumber/index.ts:33]:
```suggestion
    { url: example2, caption: 'With a title' },
```

File: plugins/legacy-plugin-chart-histogram/src/index.js [hunk 3]
```diff
@@ -18,11 +18,19 @@
...
+import example1 from './images/example1.jpg';
+import example2 from './images/example2.jpg';
+import example3 from './images/example3.jpg';
 import thumbnail from './images/thumbnail.png';
 import controlPanel from './controlPanel';
 
 const metadata = new ChartMetadata({
   description: '',
+  exampleGallery: [
+    { url: example1, caption: 'Population age data' },
```
Comment (rusackas) [inline plugins/legacy-plugin-chart-histogram/src/index.js:30]: t()

Comment (suddjian) [inline plugins/legacy-plugin-chart-histogram/src/index.js:30]:
```suggestion
    { url: example1, caption: t('Population age data') },
```

File: plugins/legacy-preset-chart-nvd3/src/Bar/index.js [hunk 4]
```diff
@@ -18,13 +18,22 @@
...
 import controlPanel from './controlPanel';
 
 const metadata = new ChartMetadata({
+  category: t('Composition'),
   credits: ['http://nvd3.org'],
   description: 'A bar chart where the x axis is time',
+  exampleGallery: [
+    { url: grouped, caption: t('Grouped style') },
+    { url: stacked, caption: t('Stacked style') },
+    { url: percentage, caption: t('Percentage style') },
```
Comment (rusackas) [inline plugins/legacy-preset-chart-nvd3/src/Bar/index.js:35]:
```suggestion
    { url: percentage, caption: t('Contribution mode) },
```

**Review by rusackas — APPROVED** [review]
LGTM

File: plugins/legacy-preset-chart-nvd3/src/Bar/index.js [hunk 5]
```diff
@@ -18,13 +18,22 @@
...
 import controlPanel from './controlPanel';
 
 const metadata = new ChartMetadata({
+  category: t('Composition'),
   credits: ['http://nvd3.org'],
   description: 'A bar chart where the x axis is time',
+  exampleGallery: [
+    { url: grouped, caption: t('Grouped style') },
+    { url: stacked, caption: t('Stacked style') },
+    { url: percentage, caption: t('Contribution mode) },
```
Comment (rusackas) [inline plugins/legacy-preset-chart-nvd3/src/Bar/index.js:35]:
```suggestion
    { url: percentage, caption: t('Contribution mode') },
```
unterminated string constant

File: plugins/legacy-preset-chart-big-number/src/BigNumber/index.ts [hunk 6]
```diff
@@ -19,10 +19,19 @@
...
 import transformProps, { BigNumberChartProps, BigNumberFormData } from './transformProps';
+import example1 from './images/BigNumber.jpg';
+import example2 from './images/BigNumber2.jpg';
 import thumbnail from './images/thumbnail.png';
 
 const metadata = new ChartMetadata({
-  description: '',
+  category: t('Time Series'),
+  description: t(
+    'A single number accompanied by a simple line chart, to call attention to an important metric along with its change over time or other dimension. If you find yourself using a lot of big numbers on a dashboard, a more visual chart may help you make your dashboard more informative.',
```
Comment (michael-s-molina) [inline plugins/legacy-preset-chart-big-number/src/BigNumber/index.ts:29]:
```suggestion
    'A single number accompanied by a simple line chart, to call attention to an important metric along with its change over time or another dimension. If you find yourself using a lot of big numbers on a dashboard, a more visual chart may help you make your dashboard more informative.',
```

File: plugins/legacy-preset-chart-nvd3/src/Bullet/index.js [hunk 7]
```diff
@@ -22,9 +22,13 @@ import thumbnail from './images/thumbnail.png';
 import controlPanel from './controlPanel';
 
 const metadata = new ChartMetadata({
+  category: t('Relationship'),
   credits: ['http://nvd3.org'],
-  description: '',
+  description: t(
+    'Visualize three dimensions of data in a single chart (x-axis, y-axis, and bubble size).',
```
Comment (michael-s-molina) [inline plugins/legacy-preset-chart-nvd3/src/Bullet/index.js:28]:
```suggestion
    'Visualize three dimensions of data in a single chart: x-axis, y-axis, and bubble size.',
```

**Review by michael-s-molina — APPROVED** [review]
LGTM. Few non-blocking suggestions.

**Pull request merged**